SmellDetectorTests/Unit/SmellDetectorServiceTest.cs

```diff
@@ -71,5 +71,59 @@ public void Generate_Smell_Detection_Report_Without_Issues()
             report.Report.Count().ShouldBe(expectedIssues);
         }
 
+
+        [Fact]
+        public void Generate_Smell_Detection_Report_For_GodClass_Issue1()
+        {
+            DetectionService detectionService = new DetectionService();
```

Comment:
It's good enough for now, but I would prefer if we used the InlineMember annotation (see https://www.youtube.com/watch?v=rjbip2JWqZk) and the Test Data Factory pattern (see https://github.com/Clean-CaDET/platform-repository-compiler/blob/master/RepositoryCompilerTests/DataFactories/CodeFactory.cs) to have one test instead of three.

Reply:
I think I’m done with refactoring, take a look just in case